mycore-restapi/src/main/java/org/mycore/restapi/MCRTransactionFilter.java

```diff
@@ -42,7 +41,6 @@ public void filter(ContainerRequestContext requestContext) {
         if (MCRSessionMgr.isLocked()) {
             return;
         }
-        MCRSession mcrSession = MCRSessionMgr.getCurrentSession();
```

Comment:
do not remove this method call, 
because MCRSessionMgr.getCurrentSession(); binds the MCRSession to the local thread

but you can skip saving the method result in a variable.